# Disables OpenAL Soft's libsndfile support.
# When configured with libsndfile support, OpenAL Soft fails to compile for 32-bit ARM versions of Ubuntu 22.04.

--- a/utils/meson.build
+++ b/utils/meson.build
@@ -8,7 +8,7 @@ utils = {
 }
 
 sndfile_dep = dependency(
-  'sndfile',
+  '',
   required: false,
 )
 if sndfile_dep.found()
